docs/wrapping-react/more-wrapping-examples.md

```diff
@@ -0,0 +1,430 @@
+10|         // Data: Data to be displayed in the chart
+11|         data: [
+12|             \{ month: 'Jan', avgTemp: 2.3, iceCreamSales: 162000 },
+13|             \{ month: 'Mar', avgTemp: 6.3, iceCreamSales: 302000 },
+14|             \{ month: 'May', avgTemp: 16.2, iceCreamSales: 800000 },
+15|             \{ month: 'Jul', avgTemp: 22.8, iceCreamSales: 1254000 },
+16|             \{ month: 'Sep', avgTemp: 14.5, iceCreamSales: 950000 },
+17|             \{ month: 'Nov', avgTemp: 8.9, iceCreamSales: 200000 },
+18|         ],
+19|         // Series: Defines which chart type and data to use
+20|         series: [\{ type: 'bar', xKey: 'month', yKey: 'iceCreamSales' }],
+21|     });
+22| 
+23|     // React Chart Component
+24|     return (
+25|         // AgCharts component with options passed as prop
+26|         <AgCharts options=\{chartOptions} />
+27|     );
+28| }
+29| 
+30| // Render component inside root element
+31| const root = ReactDOM.createRoot(document.getElementById('root'));
+32| root.render(<ChartExample />);
+```
+
+
+
+```python
+import reflex as rx
+from reflex import Component, Var
+
+class AgCharts(Component):
+    """ A simple line chart component using AG Charts """
+
+    library = "ag-charts-react"
+    
+    tag = "AgCharts"
+
+    options: Var[dict]
+
+
+chart = AgCharts.create
+
+
+class State(rx.State):
+    """The app state."""
+    chart_options: dict = {
+           "data": [
+                \{"month":"Jan", "avgTemp":2.3, "iceCreamSales":162000},
+                \{"month":"Mar", "avgTemp":6.3, "iceCreamSales":302000},
+                \{"month":"May", "avgTemp":16.2, "iceCreamSales":800000},
+                \{"month":"Jul", "avgTemp":22.8, "iceCreamSales":1254000},
+                \{"month":"Sep", "avgTemp":14.5, "iceCreamSales":950000},
+                \{"month":"Nov", "avgTemp":8.9, "iceCreamSales":200000}
+            ],
+            "series": [\{"type":"bar", "xKey":"month", "yKey":"iceCreamSales"}]
+        }
+
+def index() -> rx.Component:
+    return chart(
+        options=State.chart_options,
+    )
+
+app = rx.App()
+app.add_page(index)
+```
+
+
+## React Leaflet
+
+```python exec
+from pcweb.pages import docs
+```
+
+In this example we are wrapping the React Leaflet library from this [NPM package](https://www.npmjs.com/package/react-leaflet).
+
+On line `1` we import the `dynamic` function from Next.js and on line `21` we set `ssr: false`. Lines `4` and `6` use the `dynamic` function to import the `MapContainer` and `TileLayer` components from the `react-leaflet` library. This is done in Reflex by using the `NoSSRComponent` class when defining the component. This is used to dynamically import the `MapContainer` and `TileLayer` components from the `react-leaflet` library. This is done in Reflex by using the `NoSSRComponent` class when defining the component. There is more information of when this is needed on the `Dynamic Imports` section of this [page]({docs.wrapping_react.guide.path}).
```

Comment:
duplicate sentence "This is done in Reflex by using the `NoSSRComponent` class when defining the component."

Reply:
removed